### PR TITLE
book controller added

### PR DESCRIPTION
**Pull request by AndreyDMA**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -60,36 +67,60 @@
             <artifactId>jaxb-api</artifactId>
             <version>2.3.1</version>
         </dependency>
+
+        <!-- JSTL Dependency-->
```
Comment (Sofasmile) [inline pom.xml]: do we need comments?

File: pom.xml [hunk 2]
```diff
@@ -106,7 +137,6 @@
                     <linkXRef>false</linkXRef>
                 </configuration>
             </plugin>
-
         </plugins>
     </build>
 </project>
```
Comment (Sofasmile) [inline pom.xml]: add a blank line at the end of the file

File: src/main/java/mate/academy/spring/dao/BookDao.java [hunk 3]
```diff
@@ -10,5 +10,7 @@
 
     List<Book> findByName(String name);
 
+    Book getById(Long id);
```
Comment (Sofasmile) [inline src/main/java/mate/academy/spring/dao/BookDao.java]: we can use optional)

File: src/main/java/mate/academy/spring/entity/Author.java [hunk 4]
```diff
@@ -66,4 +63,6 @@ public void setSurname(String surname) {
     public void setBooks(List<Book> books) {
         this.books = books;
     }
+
```
Comment (Sofasmile) [inline src/main/java/mate/academy/spring/entity/Author.java]: remove two lines

File: src/main/webapp/WEB-INF/views/allBooks.jsp [hunk 5]
```diff
@@ -0,0 +1,65 @@
+<%--
```
Comment (Sofasmile) [inline src/main/webapp/WEB-INF/views/allBooks.jsp]: do we need comments in jsp?

File: src/main/java/mate/academy/spring/controller/InjectDataController.java [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+@RequestMapping("/injectData")
+public class InjectDataController {
+
+    @GetMapping
```
Comment (Sofasmile) [inline src/main/java/mate/academy/spring/controller/InjectDataController.java]: suggestion: we can use annotation PostConstruct for injecting data when we start this application

File: pom.xml (hunk 2 above)
Comment (Sofasmile) [reply pom.xml]: add a blank line!

File: src/main/java/mate/academy/spring/dao/BookDao.java (hunk 3 above)
Comment (AndreyDMA) [reply src/main/java/mate/academy/spring/dao/BookDao.java]: I'll do it at weekend))

File: src/main/java/mate/academy/spring/entity/Author.java (hunk 4 above)
Comment (AndreyDMA) [reply src/main/java/mate/academy/spring/entity/Author.java]: done

File: src/main/webapp/WEB-INF/views/allBooks.jsp (hunk 5 above)
Comment (AndreyDMA) [reply src/main/webapp/WEB-INF/views/allBooks.jsp]: fixed

File: src/main/java/mate/academy/spring/controller/InjectDataController.java (hunk 6 above)
Comment (AndreyDMA) [reply src/main/java/mate/academy/spring/controller/InjectDataController.java]: agree

**Pull request merged**